torch/autograd/functional.py

```diff
@@ -1,11 +1,24 @@
 import torch
 from typing import Tuple, List
+from . import forward_ad as fwAD
+from torch._vmap_internals import _vmap  # can we replace with use of is_grads_batched=True
```

Comment:
Not sure wht the comment means?

Reply:
Good catch, not sure why that is still here. It was from an earlier PR in the stack. I'm removing it.